api/observation.go

```diff
@@ -0,0 +1,410 @@
+package api
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/ONSdigital/dp-graph/v2/observation"
+	errs "github.com/ONSdigital/dp-observation-api/apierrors"
+	"github.com/ONSdigital/dp-observation-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+// Upper limit, if this is not big enough, we may need to consider increasing value
+// and then if this has a performance hit then consider paging
+const (
+	defaultObservationLimit = 10000
```

Comment:
If this is likely to change I think it make sense to make this a config value now.